sql/core/src/test/scala/org/apache/spark/sql/connector/DeleteFromTableSuiteBase.scala

```diff
@@ -269,7 +269,7 @@ abstract class DeleteFromTableSuiteBase extends RowLevelOperationSuiteBase {
            |WHERE
            | id NOT IN (SELECT * FROM deleted_id WHERE value IS NOT NULL)
            | AND
-           | EXISTS (SELECT 1 FROM FROM deleted_dep WHERE dep = deleted_dep.value)
+           | EXISTS (SELECT 1 FROM deleted_dep WHERE dep = deleted_dep.value)
```

Comment:
Is worth to change the behaviour, and enforce users to modify their code because of trailing commas? I doubt about it.